caproto/curio/client.py

```diff
@@ -249,10 +249,18 @@ def __init__(self):
         for host in ca.get_address_list():
             if ':' in host:
                 host, _, specified_port = host.partition(':')
-                await self.udp_sock.sendto(bytes_to_send,
-                                           (host, int(specified_port)))
+                try:
```

Comment:
I know you're following best practices here and keeping the `try` portion small, but I'd consider fudging the rules a bit here and putting both of these `sendto`s in the same `try` block.

Reply:
Fair. I considered it and almost did that. Your vote tips the balance.